### PR TITLE
docs(node): add recipes for various databases w/ node

### PR DESCRIPTION
**Pull request by barbados-clemens**
add new 'database' section that for the node examples with dbs

unsure if any extra info should go in the docs since the repo REAME covers what would be said.

### CODE REVIEW

File: docs/generated/manifests/menus.json [hunk 1]
```diff
@@ -2730,6 +2730,31 @@
         "children": [],
         "disableCollapsible": false
       },
+      {
+        "name": "Database",
```
Comment (barbados-clemens) [inline docs/generated/manifests/menus.json:2734]: unsure if this should be the heading or not, also should this be plural "Databases"?

File: docs/map.json [hunk 2]
```diff
@@ -1192,6 +1192,19 @@
             }
           ]
         },
+        {
+          "name": "Database",
```
Comment (barbados-clemens) [inline docs/map.json:1196]: unsure if this should be the heading or not, also should this be plural "Databases"?

**Comment by isaacplmann** [comment]
It doesn't make sense to make a new category with only one item in it.  I would put the recipe under other unless you can think of another category that makes sense with more recipes.

**Comment by barbados-clemens** [comment]
> It doesn't make sense to make a new category with only one item in it. I would put the recipe under other unless you can think of another category that makes sense with more recipes.

@isaacplmann there will be more to come. 

so maybe we just hold off on this PR until they are more ready?
currently [fastify+postgres](https://github.com/nrwl/nx-recipes/pull/22) is PR'd with plans for others though the Node Examples effort.

**Comment by barbados-clemens** [comment]
> > It doesn't make sense to make a new category with only one item in it. I would put the recipe under other unless you can think of another category that makes sense with more recipes.
> 
> @isaacplmann there will be more to come.
> 
> so maybe we just hold off on this PR until they are more ready? currently [fastify+postgres](https://github.com/nrwl/nx-recipes/pull/22) is PR'd with plans for others though the Node Examples effort.


updated with all the recipes that were made for databases with node.

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**